### ISSUE TITLE
npm run dev not working

### ISSUE THREAD

**Issue by yujong-lee** [marcomafessolli/remix-prisma-cloudflare-workers#1]
Hi @marcomafessolli. Thanks for sharing your work.
I was struggling setting up for remix-prisma-cloudflare-workers, and your repo is right for me.

However, I have problem with using it.

Here's what I did.

1. Clone the repo.
2. npm install
3. npm run dev

Here's error message.  Can you help me go through this?

Thank you.

<img width="1816" alt="스크린샷 2022-01-13 오후 3 30 29" src="https://user-images.githubusercontent.com/61503739/149277908-302de3b5-177b-491d-9aee-afb5a3da5301.png">
<img width="1802" alt="스크린샷 2022-01-13 오후 3 30 37" src="https://user-images.githubusercontent.com/61503739/149277911-4425c236-c354-4563-aa16-d1aad52a5f22.png">
<img width="1801" alt="스크린샷 2022-01-13 오후 3 30 45" src="https://user-images.githubusercontent.com/61503739/149277914-c591f526-d84f-4995-8c8c-405f8a89d4a5.png">

**Comment by marcomafessolli** [comment]
Hey @yujong-lee , what node version are you on? I remember having some issues on node>16

**Comment by marcomafessolli** [comment]
@yujong-lee so I just downloaded this repo and I was able to run it. Make sure you have the following:

* `node >= 16`
* A `.env` file with `DATABASE_URL` that points to your prisma data proxy
* You are generating your prisma client running `PRISMA_CLIENT_ENGINE_TYPE=dataproxy npx prisma generate`

**Comment by marcomafessolli** [comment]
@yujong-lee I've added more details on the Readme `setup` section. Let me know if that helps

**Comment by yujong-lee** [comment]
@marcomafessolli  Thank you for help. 🙇. I really appreciate it.

My node version is v17.3.1.

I was able to solve errors with `dev` command with your new README.  (Error in screenshots above was due to not running `PRISMA_CLIENT_ENGINE_TYPE=dataproxy npx prisma generate`.)

But I have new question.

> > $ npm run dev
> 
> And then check http://127.0.0.1:8787. You're ready 

Is `npm run dev` is enough for running app in localhost? (not working for me)

Here's `package.json` when initialize app with `npx create-remix (and Cloudflare option)`.

```
    "build": "remix build",
    "dev": "remix watch",
    "postinstall": "remix setup cloudflare-workers",
    "build:worker": "esbuild --define:process.env.NODE_ENV='\"production\"' --minify --bundle --sourcemap --outdir=dist ./worker",
    "dev:worker": "esbuild --define:process.env.NODE_ENV='\"development\"' --bundle --sourcemap --outdir=dist ./worker",
    "start": "miniflare --build-command \"npm run dev:worker\" --watch",
    "deploy": "npm run build && wrangler publish"
```



Something closest to `npm run start` in your `package.json` is :
```
"dev:worker": "miniflare --build-command \"node ./scripts/build.js\" --watch"
```

So I run it, got error message :

<img width="862" alt="스크린샷 2022-01-18 오후 1 06 25" src="https://user-images.githubusercontent.com/61503739/149869224-057e3bde-3094-4799-8994-4551582acda6.png">

**Comment by marcomafessolli** [comment]
@yujong-lee `npm run dev` should be all you need locally, what happens when you run it? Also, are you using any other service? It looks like `aws-sdk` could be the problem

**Comment by yujong-lee** [comment]
@marcomafessolli  

Actually I was trying to use your template with my existing code to deploy. But It looks like something unrelated to your template causes the problem. 

After removing all extra code I copied, it works. https://github.com/marcomafessolli/remix-prisma-cloudflare-workers/issues/1#issuecomment-1014931883 This was helpful.

Previously, for some reason(maybe `aws-sdk`) `npm run dev` execute only `npm run build:remix` part, not `concurrently \"npm:dev:*\"` which runs app locally. ( I was not familar to `npm:dev:*` command, so I messed up explaining my situation in https://github.com/marcomafessolli/remix-prisma-cloudflare-workers/issues/1#issuecomment-1015053302)

Now that I've confirmed that it's working in a small example, I think I can solve the problems one by one by moving the codes I've already written to this template.

Thanks for your kind, quick response.
